.github/README.md

```diff
@@ -333,15 +331,15 @@ To verify that S3 & CloudFront are working correctly.
 3. Click on `App registrations`.
 4. Click on `New registration`.
 5. Fill in the form (adjust to the environment):
-   - Name: `justicedigital-centraldigital-intranet-development`
+   - Name: `justicedigital-centraldigital-intranet-staging`
    - Supported account types: `Accounts in this organizational directory only`
-   - Redirect URI: `Web` and `http://localhost/oauth2?action=callback`  
-     or `https://dev.intranet.justice.gov.uk/oauth2?action=callback` etc.
+   - Redirect URI: `Web` and `https://staging.intranet.justice.gov.uk/oauth2/callback`  
+     or `https://intranet.justice.gov.uk/oauth2/callback` etc.
 6. Copy the `Application (client) ID` and `Directory (tenant) ID` values,
   make them available as environment variables `OAUTH_CLIENT_ID`, `OAUTH_TENANT_ID`.
 7. Click on `Certificates & secrets` > `New client secret`.
 8. Fill in the form:
-   - Description: `Local-Intranet-v2`
+   - Description: `Staging-Intranet-v2`
```

Comment:
Hey 😄 
A quick Q... why are we using v2, do we know what does it means? 
... maybe we can drop the v all together?

Reply:
Good point, it was to differentiate between Tactical Product's secrets, but I think it's unnecessary - I'll remove.